docs/changelog.rst

```diff
@@ -16,6 +16,7 @@ Fixed
 
 - Test server generated wrong schema when ``endpoints`` option is passed via CLI. `#173`_
 - Error message if schema is not found in CLI. `#172`_
+- Show falsifying examples in case of hypothesis errors `#137`_
```

Comment:
Please, rephrase. It should be about handling Hypothesis errors. Falsifying examples are not that relevant, the important part is that the test suite doesn't stop on such errors